### PR TITLE
docs(templates): Branding guide violation fix

### PR DESCRIPTION
**Pull request by wizardofhogwarts**

### CODE REVIEW

File: packages/generators/templates/template.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html>
 	<head>
 		<meta charset="utf-8" />
-		<title>Webpack App</title>
+		<title>The webpack App</title>
```
[the remark this answers is not in the document]
Comment (wizardofhogwarts) [reply packages/generators/templates/template.html]: > Please remove "The", it's an unnecessary addition.

@rishabh3112 i think the makes the title more consistent

**Comment by wizardofhogwarts** [comment]
> Hi @wizardofhogwarts!
> Please change title such that it suggests branding guide violation.
> Also left a suggestion.

@rishabh3112 Can you please suggest a title

File: packages/generators/templates/template.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (rishabh3112) [reply packages/generators/templates/template.html]: I don't think so, To which is it consistent with?

**Comment by wizardofhogwarts** [comment]
> > Hi @wizardofhogwarts!
> > Please change title such that it suggests branding guide violation.
> > Also left a suggestion.
> 
> @rishabh3112 Can you please suggest a title

@rishabh3112 I have changed the title

**Pull request merged**